calculator_simple/src/app_window.cpp

```diff
@@ -0,0 +1,307 @@
+   btn_num_c_obj.signal_clicked().connect(sigc::mem_fun(*this, &AppWindow::handle_btn_c), false);
+   btn_num_d_obj.signal_clicked().connect(sigc::mem_fun(*this, &AppWindow::handle_btn_d), false);
+   btn_num_e_obj.signal_clicked().connect(sigc::mem_fun(*this, &AppWindow::handle_btn_e), false);
+   btn_num_f_obj.signal_clicked().connect(sigc::mem_fun(*this, &AppWindow::handle_btn_f), false);
+
+   // btn_op_add_obj.signal_clicked().connect(sigc::mem_fun(*this, &AppWindow::handle_op_add));
+   // btn_op_sub_obj.signal_clicked().connect(sigc::mem_fun(*this, &AppWindow::handle_op_sub));
+   // btn_op_mul_obj.signal_clicked().connect(sigc::mem_fun(*this, &AppWindow::handle_op_mul));
+   // btn_op_div_obj.signal_clicked().connect(sigc::mem_fun(*this, &AppWindow::handle_op_div));
+   // btn_op_mod_obj.signal_clicked().connect(sigc::mem_fun(*this, &AppWindow::handle_op_mod));
+
+   radix_slider_obj.set_digits(0);
+   radix_slider_obj.set_has_origin(true);
+
+   radix_slider_obj.signal_change_value().connect(sigc::mem_fun(*this, &AppWindow::handle_radix_change), true);
+
+   layout_buttons.attach(radix_label_obj, 0, 0, 1, 1);
+   layout_buttons.attach(radix_slider_obj, 0, 1, 1, 4);
+
+   layout_buttons.attach(btn_num_0_obj, 1, 0, 1, 1);
+   layout_buttons.attach(btn_num_1_obj, 2, 0, 1, 1);
+   layout_buttons.attach(btn_num_2_obj, 3, 0, 1, 1);
+   layout_buttons.attach(btn_num_3_obj, 4, 0, 1, 1);
+   layout_buttons.attach(btn_op_add_obj, 5, 0, 2, 1);
+
+   layout_buttons.attach(btn_num_4_obj, 1, 1, 1, 1);
+   layout_buttons.attach(btn_num_5_obj, 2, 1, 1, 1);
+   layout_buttons.attach(btn_num_6_obj, 3, 1, 1, 1);
+   layout_buttons.attach(btn_num_7_obj, 4, 1, 1, 1);
+   layout_buttons.attach(btn_op_sub_obj, 5, 1, 2, 1);
+
+   layout_buttons.attach(btn_num_8_obj, 1, 2, 1, 1);
+   layout_buttons.attach(btn_num_9_obj, 2, 2, 1, 1);
+   layout_buttons.attach(btn_num_a_obj, 3, 2, 1, 1);
+   layout_buttons.attach(btn_num_b_obj, 4, 2, 1, 1);
+   layout_buttons.attach(btn_op_mul_obj, 5, 2, 2, 1);
+
+   layout_buttons.attach(btn_num_c_obj, 1, 3, 1, 1);
+   layout_buttons.attach(btn_num_d_obj, 2, 3, 1, 1);
+   layout_buttons.attach(btn_num_e_obj, 3, 3, 1, 1);
+   layout_buttons.attach(btn_num_f_obj, 4, 3, 1, 1);
+   layout_buttons.attach(btn_op_div_obj, 5, 3, 2, 1);
+
+   layout_buttons.attach(btn_alt_dot_obj, 1, 4, 1, 1);
+   layout_buttons.attach(btn_alt_inv_obj, 4, 4, 1, 1);
+
+   layout_display.pack_start(display_area);
+
+   layout_main.pack_start(layout_display);
+   layout_main.pack_start(layout_buttons);
+
+   add(layout_main);
+
+   show_all_children();
+}
+
+/**
+ * @brief   AppWindow Destructor.
+ */
+AppWindow::~AppWindow()
+{
+}
+
+
+/**
+ * @brief   AppWindow Constructor.
+ */
+void AppWindow::handle_display_update()
+{
+   return;
+}
+
+
+/**
+ * @brief   AppWindow Constructor.
+ */
+bool AppWindow::handle_radix_change(const Gtk::ScrollType & scroll_type, const double & radix_new)
```

Comment:
**Consistent Const** failure 👻 

https://isocpp.org/wiki/faq/const-correctness#const-ref-alt

```suggestion
bool AppWindow::handle_radix_change(Gtk::ScrollType const & scroll_type, double const & radix_new)
```

Reply:
Resolved in 6b16b80